sw/device/lib/crypto/impl/ecc/p256.h

```diff
@@ -278,6 +279,21 @@ status_t p256_ecdh_finalize(p256_ecdh_shared_key_t *shared_key);
 OT_WARN_UNUSED_RESULT
 status_t p256_sideload_ecdh_start(const p256_point_t *public_key);
 
+/**
+ * Finish an async ECDH/P-256 shared key generation operation on OTBN.
+ *
+ * Uses a private key generated from a key manager seed. The key manager should
+ * already have sideloaded the key into OTBN before this operation is called.
+ *
+ * Blocks until OTBN is idle. May be used after either `p256_ecdh_start` or
+ * `p256_sideload_ecdh_start`; the operation is the same.
```

Comment:
The instruction count checks are now different between these two; the second sentence here should probably be removed both in this doc comment and in the one for `p256_ecdh_finalize`.